buildroot/share/PlatformIO/boards/BigTree_Btt002.json

```diff
@@ -16,17 +16,15 @@
     ],
     "ldscript": "stm32f407xg.ld",
     "mcu": "stm32f407vet6",
-    "variant": "BIGTREE_GENERIC_STM32F407_5X"
+    "variant": "BIGTREE_TBD"
```

Comment:
"To Be Done"?

Reply:
To be determined.
It's an unsold yet board. We don't know the real name